core_types/src/eav.rs

```diff
@@ -9,9 +9,10 @@ use crate::{
     error::{HcResult, HolochainError},
     json::JsonString,
 };
+use chrono::offset::Utc;
 use objekt;
 use std::{
-    collections::HashSet,
+    collections::BTreeSet,
```

Comment:
why BTreeSet instead of HashSet?

Reply:
Was a decision based on preserving order, two eavis won't have the same hash so we need a datastructure that implements an ordered trait to preserve order because of the index

Reply:
kk